controllers/mustgather/mustgather_controller.go

```diff
@@ -192,15 +145,12 @@ func (r *ReconcileMustGather) Reconcile(request reconcile.Request) (reconcile.Re
 		return reconcile.Result{}, err
 	}
 
-	if ok, err := r.IsValid(instance); !ok {
-		return r.ManageError(instance, err)
-	}
-
```

Comment:
I'm guessing that the removal of this, as well as the other commented out calls to `r.ManageError()` throughout this function, were a result of when we were trying to migrate this without embedding `ReconcileBase` in the `ReconcileMustGather` struct. But now that it's doing that, all these bits of code can/should probably be restored. One main motivating factor is because `ManageError` will, if I'm remembering correctly, record the error in the `MustGather` CR, which is useful functionality that is otherwise lost.

Reply:
The IsValid was removed because it doesn't actually do anything as it is supposed to be overridden with our own implementation https://github.com/redhat-cop/operator-utils/blob/v1.3.4/pkg/util/reconciler.go#L74

Thanks for the catch on the ManageError, they have been restored